phoenix-core/src/it/java/org/apache/phoenix/end2end/BackwardCompatibilityIT.java

```diff
@@ -437,8 +472,17 @@ public void run() {
     }
 
     // Executes the SQL commands listed in the given operation file from the sql_files directory
-    private void executeQueriesWithCurrentVersion(String operation) throws Exception {
+    private void executeQueriesWithCurrentVersion(String operation,
+            boolean setMaxLookBackAge) throws Exception {
         Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        if (setMaxLookBackAge) {
+            props.put(
+                QueryServices.GLOBAL_INDEX_ROW_AGE_THRESHOLD_TO_DELETE_MS_ATTRIB,
```

Comment:
why do we need to set this property?

Reply:
My bad, this is not needed, it came as part of copy from existing test. Removed it.
Thanks